libs/tx-gmail/src/convert.ts

```diff
@@ -51,7 +53,8 @@ const findHeader = (email: gmail_v1.Schema$Message, header: string) =>
 function getAddressCoin(email: gmail_v1.Schema$Message) {
   const toField = findHeader(email, "To");
   if (toField) {
-    const match = /<([A-Fx0-9]+)@thecoin.io>\s*$/gi.exec(toField);
+    const re = new RegExp(`([A-Fx0-9]+)@${DEPOSIT_DOMAIN}`, "gi")
```

Comment:
Our deposits can be differentiated by the host alias.  Our regular deposits come in on `0x1234@thecoin.io`, and testing deposits on `0x1234@test.thecoin.io`.